lib/syskit/network_generation/engine.rb

```diff
@@ -703,19 +703,31 @@ def compute_system_network(
                     Engine.discover_requirement_tasks_from_plan(real_plan),
                 garbage_collect: true,
                 validate_abstract_network: true,
-                validate_generated_network: true
+                validate_generated_network: true,
+                default_deployment_group: nil,
+                validate_deployed_network: false,
+                early_deploy: false
             )
                 requirement_tasks = requirement_tasks.to_a
                 instance_requirements = requirement_tasks.map(&:requirements)
                 system_network_generator = SystemNetworkGenerator.new(
-                    work_plan, event_logger: event_logger, merge_solver: merge_solver
-                )
-                toplevel_tasks = system_network_generator.generate(
-                    instance_requirements,
-                    garbage_collect: garbage_collect,
-                    validate_abstract_network: validate_abstract_network,
-                    validate_generated_network: validate_generated_network
+                    work_plan,
+                    event_logger: event_logger,
+                    merge_solver: merge_solver,
+                    default_deployment_group: default_deployment_group,
+                    early_deploy: early_deploy,
+                    validate_deployed_network: validate_deployed_network
```

Comment:
I don't think it makes sense to have `validate_deployed_network` here if all the `validate_*` options are passed to `generate`